releasenotes/notes/lo-freqs-circuit-c2ec161873eb4a68.yaml

```diff
@@ -0,0 +1,29 @@
+---
+features:
+  - |
+    The ability to pass qubit and measurement LO frequencies has now been added to circuit jobs. You
+    can use the fields ``qubit_lo_freq`` and ``meas_lo_freq`` as arguments to either ``execute`` or
+    ``assemble`` with circuit and pulse jobs. The fields are lists of floats representing
+    frequencies in Hz and are indexed by qubit number (entry 0 is qubit 0, entry 1 is qubit 1,
+    etc...). The length of the list must be equal to the number of qubits in the system. For
+    example, consider running circuit ``circ`` on some ``backend`` with 3 qubits. You want to use
+    qubit frequencies ``[5e9Hz, 4.9e9Hz, 5.1e9Hz]`` and measurement frequencies
+    ``[5.5e9Hz, 6.5e9Hz, 7.1e9Hz]``. This can be achieved by::
```

Comment:
```suggestion
    qubit frequencies ``[5e9, 4.9e9, 5.1e9]`` and measurement frequencies
    ``[5.5e9, 6.5e9, 7.1e9]``. This can be achieved by::
```

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6167/commits/053e8d973a22491a48f519dbc77c85879b74d90b.